src/app-logic.js

```diff
@@ -24,7 +25,10 @@ export default function useAppLogic() {
 
     return proposals.reduce(
       ({ myStakes }, proposal) => {
-        if (proposal.executed || !proposal.stakes) {
+        if (
+          proposal.status === PROPOSAL_STATUS_EXECUTED_STRING ||
```

Comment:
A proposal could also be cancelled.

```suggestion
          !proposal.status === PROPOSAL_STATUS_ACTIVE_STRING
```

Reply:
but we don't want to count the stakes on cancelled proposals?